### DISCUSSION TITLE
Azure WebApp provides an FTP username that contains special characters \$ that SFTP does not accept

### DISCUSSION THREAD

**Discussion by ruedigerbr** [Natizyskunk/vscode-sftp#317] [Q&A] [answered]
Azure provides a username for accessing a webapp via FTP that cannot be modified. The problem is that this username contains special characters, in this case a back slash followed by a dollar sign \$. Entering this as a username into the .json file to configure the FTP access to Azure produces an error message. Is there a way to make SFTP accept this username?

![azure_username](https://user-images.githubusercontent.com/113355682/220655685-528dea4e-574d-4199-8654-92ecbca76091.JPG)

**Comment by satiromarra** [comment, marked as answer]
@ruedigerbr Use a double escape like: `blablabla\\$blobloblo`

**Comment by ruedigerbr** [comment]
Hi Natan,
Thanks!!

________________________________
